test/Unit/Runners/PHPUnit/OptionsTest.php

```diff
@@ -346,7 +349,7 @@ public function testProvidedOptions(): void
         ], $options->filtered());
 
         static::assertTrue($options->hasLogTeamcity());
-        static::assertTrue($options->hasCoverage());
+        static::assertFalse($options->hasCoverage());
```

Comment:
If you change this, there are no place where we test `static::assertTrue($options->hasCoverage());`, can you rearrange test to have both cases please?

Reply:
On it !

Reply:
I just saw that it is tested in `testGatherOptionsFromConfiguration` https://github.com/paratestphp/paratest/pull/584/files#diff-ed06b3688f45d43818a780f94b65eb6bfdb6781cbe77f15d2135cfd4c4b146a6R387

Do I still need to add another test when coverage is enabled from command lines options ?

Reply:
Yes: `testGatherOptionsFromConfiguration` tests, as it says, options from configuration.

We need instead both `assertTrue` and `assertFalse` over `$options->hasCoverage()` in the pure CLI options scenario.

So please revert this one, and create a separate test for the new case

Reply:
Solved by b6cd92f